Discovery/src.Docker/Fortune-Teller-Service/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM mcr.microsoft.com/dotnet/sdk:5.0-buster-slim as builder
+FROM mcr.microsoft.com/dotnet/sdk:8.0 AS builder
 WORKDIR /src
 COPY ["Fortune-Teller-Service/FortuneTeller.Service.csproj", "Fortune-Teller-Service/"]
+COPY ["Fortune-Teller-Service/Directory.Build.props", "Fortune-Teller-Service/"]
```

Comment:
Oh, it makes sense that you were talking about this sample... Yeah, I'm not surprised the dockerfile was wildly out of date, it was never a common test path and I think was more of an experiment (back before we even added Directory.Build.props) than a recommendation